src/state.rs

```diff
@@ -0,0 +1,71 @@
+use ethers::types::U256;
+use std::collections::HashMap;
+use std::vec::Vec;
+
+// Data needed to commit new block
```

Comment:
You can use [doc comments](https://doc.rust-lang.org/reference/comments.html#doc-comments) (`///`) here instead of normal